ratpack-kafka-consumer/src/main/java/smartthings/ratpack/kafka/KafkaConsumerModule.java

```diff
@@ -0,0 +1,42 @@
+package smartthings.ratpack.kafka;
+
+import com.google.inject.Scopes;
+import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import ratpack.guice.ConfigurableModule;
+
+import java.util.Properties;
+import java.util.Set;
+
+public class KafkaConsumerModule extends ConfigurableModule<KafkaConsumerModule.Config> {
+
+	protected void configure() {
+		bind(KafkaConsumerService.class).in(Scopes.SINGLETON);
+	}
+
+	public static class Config {
```

Comment:
I agree, not sure I see a value in two separate modules. Just means we have to rename things and update the downstream projects.